Entities/Common/Movement/FallDamage.as

```diff
@@ -57,7 +57,7 @@ void onCollision(CBlob@ this, CBlob@ blob, bool solid, Vec2f normal, Vec2f point
 		// stun on fall
 		const u8 knockdown_time = 12;
 
-		if (doknockdown && setKnocked(this, knockdown_time))
+		if (!this.hasTag("dead") && doknockdown && setKnocked(this, knockdown_time))
```

Comment:
This causes `FallDeath.ogg` to not be played when falling to your death (at least in singleplayer).

Reply:
I committed a different solution.

Added `onHealthChange()` to `FallDamage.as`, which saves a bool to the runner blob.
If the old health was above zero, the bool is set true, otherwise false.
After that, `onCollision()` is executed next. 
Here, the sound is played only if the bool is true.

Because 12 is the only value 'knockdown_time' will be set to, I placed that variable outside the function.

Reply:
Nvm, it's not working in online. So the solution is not good to go yet.

Reply:
I found that this original solution (commit 1) works in both localhost and dedicated server. Why the requested changes? The first commit works as intended with no issues found during my testing. Here is a video showing this first change on a dedicated server, turn on sound:

https://user-images.githubusercontent.com/68350259/202875039-a7229379-24e3-4ac1-908f-096629c1cd80.mp4

As you can hear, the dead bodies do not make any sounds while the alive do.
Unless I am missing something, return to your first solution.

Reply:
I tested and this is the result:

Prior to this PR:
- Dead body makes a grunt sound when throwing from 13 blocks height
- Alive player falling from a very high place makes a death sound (grunt or bone break)

Commit 1:
- Dead body **doesn't** make a grunt sound when throwing from 13 blocks height (Good)
- Alive player falling from very high **doesn't** make a death sound (Bad)

Commit 4:
- Dead body **doesn't** make a grunt sound when throwing from 13 blocks height (Good) 
- Alive player falling from a very high place makes a death sound (grunt or bone break) (Good)

This also applies to `!builder` spawned in.
Maybe you didn't load the patched file correctly? I can make a video if needed.